wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -395,7 +450,7 @@ newtype AddressValidity = AddressValidity { isValid :: Bool }
 deriveJSON Serokell.defaultOptions ''AddressValidity
 
 instance ToSchema AddressValidity where
-    declareNamedSchema = genericSchemaDroppingPrefix "is"
+    declareNamedSchema = genericSchemaDroppingPrefix "is" (\_ -> Prelude.id)
```

Comment:
You can use `identity` from `Universum` if you want, that's exactly the same 😁

Reply:
Ah! Thanks. Not used to `Universum` yet.